outlines/models/transformers.py

```diff
@@ -163,11 +163,13 @@ def format_output_type(self, output_type):
 
 class Transformers(Model):
```

Comment:
We may be able to do this automatically. It looks like all models implemented implemented with JAX inherit from [`FlaxPretrainedModel`](https://github.com/huggingface/transformers/blob/c9d1e5238a752813ba91a8751a638a09b5efbb73/src/transformers/modeling_flax_utils.py#L168)